src/Photos/Enums.cs

```diff
@@ -378,6 +383,8 @@ public enum PHPhotosError : long {
 		LibraryVolumeOffline = 3114,
 		RelinquishingLibraryBundleToWriter = 3142,
 		SwitchingSystemPhotoLibrary = 3143,
+		[TV (14,0), Mac (10,16), iOS (14,0)]
```

Comment:
we do not add availability to error enums
that's because they are output only (from API) and it's easier to handle values alone (than adding version checks)